requirements.txt

```diff
@@ -0,0 +1,37 @@
+ansible==9.5.1
+ansible-compat==24.5.1
+ansible-core==2.16.6
+attrs==23.2.0
+bracex==2.4
+certifi==2024.2.2
+cffi==1.16.0
+charset-normalizer==3.3.2
+click==8.1.7
+click-help-colors==0.9.4
+cryptography==42.0.7
+distro==1.9.0
+docker==7.0.0
+enrich==1.2.7
+idna==3.7
+Jinja2==3.1.4
+jsonschema==4.22.0
+jsonschema-specifications==2023.12.1
+markdown-it-py==3.0.0
+MarkupSafe==2.1.5
+mdurl==0.1.2
+molecule==24.2.1
+molecule-plugins==23.5.3
+packaging==24.0
+pluggy==1.5.0
+pycparser==2.22
+Pygments==2.18.0
+PyYAML==6.0.1
+referencing==0.35.1
+requests==2.31.0
+resolvelib==1.0.1
+rich==13.7.1
+rpds-py==0.18.1
+selinux==0.3.0
+subprocess-tee==0.4.1
+urllib3==2.2.1
+wcmatch==8.5.2
```

Comment:
What a hell to maintain :try-not-to-cry: ...

Reply:
this is generated by `pip freeze` command , i run it inside a virt env with only the required packages installed so it supposedly to be all the dependencies.